esmvalcore/_main.py

```diff
@@ -328,6 +345,9 @@ def run(recipe,
         config_file: str, optional
             Configuration file to use. If not provided the file
             ${HOME}/.esmvaltool/config-user.yml will be used.
+        resume: list(str), optional
+            Resume one or more previous runs by using preprocessor output files
+            from these directories.
```

Comment:
we need examples for inputs here, is it the recipe name?

Reply:
output directory, as it says in the text, but I'll add documentation for readthedocs too. If you have a suggestion for improving this text, that would be most welcome of course

Reply:
yes, just state the object name directly: output directory name, even better, call the option eg `output_directory_to_resume`

Reply:
That seems a bit long. Would `resume_from` or `reuse` work for you?

Reply:
yup that'll do, Piggy, that'll do :grin:

Reply:
Changed it to ``--resume_from`` now.